docs/_components/example/template.njk

```diff
@@ -8,6 +8,9 @@
 {%- endset -%}
 
 {%- set examplePath = "/examples/" + params.component -%}
+{% if params.example %}
+  {%- set examplePath = examplePath + "-" + params.example -%}
+{% endif %}
```

Comment:
In #192 I wanted to show the full code example in the Nunjucks tab, not the component isolated on its own. To achieve that, I’ve take the bit from this PR where each tab gets a unique identifier, but have avoided splitting up `component` and `example` into separate params meaning each example shows the preview and the associated code used to create it. 

If that makes sense. Can update this PR if it doesn’t!

Reply:
Not quite sure I follow this. I added the 'example' option as a quick way to have multiple examples per component, based loosely on the design system one (which has 3 options: `group`, `item` and `example`).

Maybe that's overkill though, and we could just have a single param named `example` (instead of `component`) for now, all within a single folder?

Reply:
Yeah, let’s keep it simple for now and rename the param `example`. We can split this up later, if we need to.

Reply:
Ok, done in #196